pkg/rid/cockroach/store.go

```diff
@@ -161,44 +162,35 @@ func (s *Store) Bootstrap(ctx context.Context) error {
 	return err
 }
 
-// GetVersion returns the current semver of the schema
+// GetVersion returns the Version string for the Database.
+// If the DB was is not bootstrapped using the schema manager we throw and error
 func (s *Store) GetVersion(ctx context.Context) (string, error) {
-	// We treat the existence of cells_subscriptions as running on the initial
-	// version, 1.0.0
+	// when we have multiple databases we will have to substitute
+	// defaultdb for other database names, this field is current useless
 	const query = `
 		SELECT EXISTS (
   		SELECT *
 		  FROM information_schema.tables 
-   		WHERE table_name = 'cells_subscriptions'
+		WHERE table_name = 'schema_versions'
```

Comment:
I'm debating if I should add a fall back method for determining the DB Version that was bootstrapped before Schema Manager is added. This is not hard, I would simply check the for the existence of the tables that should exists and go on from there.

Reply:
I think it's cleaner to just require the table (thereby requiring migrations to be run).  I think it's ok to have a one-time breaking change requiring users to migrate since that will be the process going foward.

Reply:
if we're doing a breaking change, let's bump the RequiredMajorSchemaVersion, and add a migration that increments that as well.